backend/app/api/endpoints/upload.py

```
@@ -306,19 +320,20 @@ async def upload_health_check():
     response_model=BatchUploadResponse,
     status_code=status.HTTP_201_CREATED,
     summary="Batch Upload Files",
-    description="Upload multiple files in a single request. Supports mixed file types.",
+    description="Upload multiple files in parallel. Supports mixed file types with partial success handling.",
 )
 async def upload_files_batch(
     current_user: Annotated[User, Depends(get_current_user)],
+    db: SessionDep,
     files: List[UploadFile] = File(..., description="List of files to upload"),
```

Ruff (0.11.9)

Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)